src/types/transactions/base.ts

```diff
@@ -182,12 +182,12 @@ export interface TransactionParams {
   /**
    * Integer first protocol round on which this txn is valid
    */
-  firstRound: number;
+  firstValid: number;
```

Comment:
Can you also rename the `reKeyTo` field to `rekeyTo`?

Reply:
good point -- [1258d1a](https://github.com/algorand/js-algorand-sdk/pull/804/commits/1258d1a6864851b6a5816de5bd3428e52171fe4c)